src/components/form/form.tsx

```diff
@@ -1,40 +1,56 @@
 /* eslint-disable @typescript-eslint/no-misused-promises */
+import React, { forwardRef, useImperativeHandle } from "react";
 import { FieldValues, FormProvider, useForm } from "react-hook-form";
-import { zodResolver } from "@hookform/resolvers/zod";
-import { FormProps } from "./form.types";
 import { Form as AntdForm } from "antd";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { FormHandle, FormProps } from "./form.types";
+
+declare module "react" {
+    function forwardRef<T, P = NonNullable<unknown>>(
+        render: (props: P, ref: React.Ref<T>) => React.ReactNode | null
+    ): (props: P & React.RefAttributes<T>) => React.ReactNode | null;
+}
```

Comment:
```suggestion
declare module "react" {
    function forwardRef<T, P = NonNullable<unknown>>(
        render: (props: P, ref: React.Ref<T>) => React.ReactNode | null
    ): (props: P & React.RefAttributes<T>) => React.ReactNode | null;
}
```

move to global type declaration